scripts/settings.js

```diff
@@ -86,15 +86,15 @@ export default class Settings {
       Constants.MODULE_ID,
       Settings.MODIFY_STATUS_EFFECTS,
       {
-        name: 'Modify Status Effects',
-        hint: 'This is how status effects on the token HUD will be modified. Replacing them means all other status effects will be removed in favor of the conditions provided by Convenient Effects. Adding them means they are appended to the end of the existing status effects. Requires a Foundry reload on change.',
+        name: game.i18n.localize("Settings.ModifyStatusEffects"),
+        hint: game.i18n.localize("Settings.ModifyStatusEffectsHint"),
         scope: 'world',
         config: true,
-        default: 'none',
+        default: game.i18n.localize("Settings.ModifyStatusEffectsNone"),

```

Comment:
Defaults should probably not be localized in cases where the settings has choices. The default uses they choice name as the key. So onAddOrRemove maps to the onAddOrRemove choice below, but what gets displayed is the value for onAddOrRemove.

So essentially, everywhere there is a default in a setting with choices, the default should not be localized.

Reply:
Noted! Will undo this and others